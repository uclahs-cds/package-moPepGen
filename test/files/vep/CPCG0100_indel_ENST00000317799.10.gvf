##fileformat=VCFv4.2
##mopepgen_version=0.0.1
##parser=parseVEP
##reference_index=/data/Index/GRCh38-EBI-GENCODE34
##genome_fasta=
##annotation_gtf=
##source=gindel
##CHROM=<Description='Gene ID'>
##INFO=<ID=TRANSCRIPT_ID,Number=1,Type=String,Description="Transcript ID">
##INFO=<ID=GENE_SYMBOL,Number=1,Type=String,Description="Gene Symbol">
##INFO=<ID=GENOMIC_POSITION,Number=1,Type=String,Description="Genomic Position">
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO
ENSG00000138029.14	11088	INDEL-11088-G-GACT	G	GACT	.	.	TRANSCRIPT_ID=ENST00000317799.10;GENOMIC_POSITION=chr2:26254257-26254258;GENE_SYMBOL=HADHB
