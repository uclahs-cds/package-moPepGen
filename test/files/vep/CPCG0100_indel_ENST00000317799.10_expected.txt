MTTILTYPFK
MTTILTYPFKNLPTASK
MTTILTYPFKNLPTASKWALR
TTILTYPFK
TTILTYPFKNLPTASK
TTILTYPFKNLPTASKWALR
